contentcuration/contentcuration/frontend/channelEdit/vuex/clipboard/actions.js

```diff
@@ -218,26 +291,29 @@ export function loadChannelColors(context) {
   }, Promise.resolve(Object.values(context.state.channelColors)));
 }
 
-/**
- * @param context
- * @param {string} node_id
- * @param {string} channel_id
- * @param {string|null} [target]
- * @param {boolean} [deep]
- * @param children
- * @return {*}
- */
-export function copy(context, { node_id, channel_id, extra_fields = {} }) {
+export function doCopy(context, { node_id, channel_id, extra_fields = {} }) {
```

Comment:
Breakout `doCopy` so `copy` and `copyAll` can handle loading channels and adding nodes themselves.